pkg/service/process_manager.go

```diff
@@ -77,8 +78,27 @@ func (s *ProcessManager) startIngress(ctx context.Context, p *params.Params, clo
 	defer span.End()
 
 	h := &process{
-		info:          p.IngressInfo,
-		sessionCloser: closeSession,
+		info: p.IngressInfo,
+		sessionCloser: func(ctx context.Context) {
+			if closeSession != nil {
+				closeSession(ctx)
+			}
+
+			go func() {
+				time.Sleep(time.Second)
+
+				s.mu.Lock()
+				h := s.activeHandlers[p.State.ResourceId]
+
+				if h != nil && !h.closed.IsBroken() && h.cmd != nil {
+					logger.Infow("killing handler process still present after termination was requested", "ingressID", h.info.IngressId, "resourceID", p.State.ResourceId, "startedAt", p.State.StartedAt)
+					if err := h.cmd.Process.Signal(syscall.SIGKILL); err != nil {
```

Comment:
Not sure if it's important, but `killAll` uses `SIGINT`, while this line uses `SIGKILL`. Probably both using `SIGKILL` in both?

Reply:
Thank you for the explanation, make sense now! Was confused by "kill" part in `killAll`.